src/user/announcements.rst

```diff
@@ -8,6 +8,21 @@ Our announcements are published to an RSS feed `here <https://conda-forge.org/do
 2023
 ----
 
+2023-08-xx: Dropping support for MacOS <10.13
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Conda-forge tries to support OS versions as long as possible, often far beyond the support of the
+OS vendor. For the last seven years, we have supported a minimal MacOS version of 10.9, despite
+the fact that Apple has released new versions on a yearly basis, and aggressively pushes its users
+to upgrade. One big reason that we have been able to do this at all, is that conda-forge is able
+to ship an up-to-date C++ standard library in its environments, but we have now reached the end of
+the road, as several key projects in the ecosystem are breaking (despite this measure) when
+compiling against such old versions of the SDK.
```

Comment:
Is there a specific policy change to mention here? "Starting on <DATE>, the default target MacOS version for new and updated conda-forge packages will be 10.13, and as time passes, it can be expected that conda-forge packages will lose compatibility with older MacOS versions."

In particular, right now the body of the announcement doesn't specifically mention 10.13 as the new minimum — that only appears in the title.